scripts/main.js

```diff
@@ -0,0 +1,124 @@
+const startBtn = document.querySelector(".start");
+const stopBtn = document.querySelector(".stop");
+const resetBtn = document.querySelector(".reset");
+const lapBtn = document.querySelector(".lap");
+const displayTime = document.querySelector(".time");
+const lapContainer = document.querySelector(".lap-container");
+const lapHeadingContainer = document.querySelector(".lap-heading-container");
```

Comment:
can you explain why all these are by querySelector, why not by id?

Reply:
ok sir i will do it with get element by id